CHANGES.rst

```diff
@@ -33,6 +33,8 @@ New Features
 
   - File name could be passed as ``Path`` object. [#4606]
 
+  - ``is_image`` property added to GroupsHDU class. [#4742]
```

Comment:
Verbiage seems okay but it is under the wrong section. Shouldn't it be in "bug fix" instead of "new feature"?